package/network/utils/comgt/files/3g.sh

```diff
@@ -72,6 +72,8 @@ proto_3g_setup() {
 					*) CODE="2,2";;
 				esac
 				export MODE="AT^SYSCFG=${CODE},3FFFFFFF,2,4"
+			elif echo "$cardinfo" | grep -q "MikroTik"; then
+				COMMAND="AT+CFUN=1" gcom -d "$device" -s /etc/gcom/runcommand.gcom || return 1
```

Comment:
`runcommand.gcom` is not included in `comgt` but in `comgt-ncm`. :confused: 

```
Sat Mar 25 18:21:22 2023 daemon.notice netifd: Interface 'mikrotik' is setting up now
Sat Mar 25 18:21:24 2023 daemon.notice netifd: mikrotik (24671): comgt 18:21:24 -> -- Error Report --
Sat Mar 25 18:21:24 2023 daemon.notice netifd: mikrotik (24671): comgt 18:21:24 -> ----> ^
Sat Mar 25 18:21:24 2023 daemon.notice netifd: mikrotik (24671): comgt 18:21:24 -> Error @0, line 1, Could not open scriptfile "/etc/gcom/runcommand.gcom".
Sat Mar 25 18:21:24 2023 daemon.notice netifd: mikrotik (24671): . (1)
Sat Mar 25 18:21:24 2023 daemon.notice netifd: mikrotik (24671):
Sat Mar 25 18:21:24 2023 daemon.notice netifd: Interface 'mikrotik' is now down
```

Reply:
Looks like it's a harmful change to move the `runcommand.gcom` script to `comgt`. :pray: